llvm/lib/Target/X86/X86FixupSetCC.cpp

```diff
@@ -136,6 +136,12 @@ bool X86FixupSetCCPass::runOnMachineFunction(MachineFunction &MF) {
           .addReg(ZeroReg)
           .addReg(Reg0)
           .addImm(X86::sub_8bit);
+
+      // Transfer the debug-instr-number to the setcc.
+      if (unsigned InstrNum = ZExt->peekDebugInstrNum())
+        ZExt->getParent()->getParent()->makeDebugValueSubstitution(
```

Comment:
I think we can just use the `MF` argument to the function rather than using get-parent twice.